pkg/query/endpointset.go

```diff
@@ -203,6 +210,9 @@ func (c *endpointSetNodeCollector) Update(nodes map[component.Component]map[stri
 	for k, v := range nodes {
 		storeNodes[k] = make(map[string]int, len(v))
 		for kk, vv := range v {
```

Comment:
I know it's not your change, but can we improve this by naming variables here? e.g `kk` is `externalLabels` and `vv` is `count`?